tests/utils/test_dataset_creation.py

```diff
@@ -158,28 +163,38 @@ def test_generate_dataset_with_external_buffer(dataset_id, env_id):
         observation, _ = env.reset()
         observations.append(observation)
 
-    # Create Minari dataset and store locally
-    dataset = minari.create_dataset_from_buffers(
-        dataset_id=dataset_id,
-        env=env,
-        buffer=buffer,
-        algorithm_name="random_policy",
-        code_permalink="https://github.com/Farama-Foundation/Minari/blob/f095bfe07f8dc6642082599e07779ec1dd9b2667/tutorials/LocalStorage/local_storage.py",
-        author="WillDudley",
-        author_email="wdudley@farama.org",
-    )
-
-    assert isinstance(dataset, MinariDataset)
-    assert dataset.total_episodes == num_episodes
-    assert dataset.spec.total_episodes == num_episodes
-    assert len(dataset.episode_indices) == num_episodes
-
-    check_data_integrity(dataset.storage, dataset.episode_indices)
-    check_env_recovery(env, dataset)
+    # Save a different environment spec for evaluation (different max_episode_steps)
+    eval_env_spec = gym.spec(env_id)
+    eval_env_spec.max_episode_steps = 123
+    eval_env = gym.make(eval_env_spec)
+    # Test for different types of env and eval_env (gym.Env, EnvSpec, and str id)
+    for env_dataset_id, eval_env_dataset_id in zip(
+        [env, env.spec, env_id], [eval_env, eval_env.spec, env_id]
+    ):
+        # Create Minari dataset and store locally
+        dataset = minari.create_dataset_from_buffers(
+            dataset_id=dataset_id,
+            env=env_dataset_id,
+            eval_env=eval_env_dataset_id,
+            buffer=buffer,
+            algorithm_name="random_policy",
+            code_permalink="https://github.com/Farama-Foundation/Minari/blob/f095bfe07f8dc6642082599e07779ec1dd9b2667/tutorials/LocalStorage/local_storage.py",
```

Comment:
not related to your PR but, can we update the link to just point to this file? Thanks